hw5/tasks/bench_test.go

```diff
@@ -0,0 +1,13 @@
+package tasks
+
+import (
+	"testing"
+)
+
+func Benchmark_MyStrToInt(b *testing.B) {
+	MyStrToInt("-9223372036854775808")
```

Comment:
this must use a for loop with `b.N` steps

Reply:
Updated.